### PR TITLE
Setup of jekyll for demoing the interconnection with GCWeb

### PR DESCRIPTION
**Pull request by duboisp**
@delisma as discussed

Here a working example that leverage GCWeb-jekyll and data-json to manage and keep the content up to date:

* https://duboisp.github.io/design-system/common-design-patterns/most-requested.html
* https://duboisp.github.io/design-system/meta/public-api.html

### CODE REVIEW

File: common-design-patterns/most-requested.html [hunk 1]
```diff
@@ -1,177 +1,41 @@
+---
+title: Most requested
```
Comment (delisma) [inline common-design-patterns/most-requested.html:2]: We need the `altLangPage` property to link to `gc-proto/design-system-demo/configurations-conception-communes/en-demande.html`

**Review by delisma — CHANGES_REQUESTED** [review]

File: common-design-patterns/most-requested.html (hunk 1 above)
Comment (delisma) [reply common-design-patterns/most-requested.html:2]: And the `dateModified` property

**Review by delisma — CHANGES_REQUESTED** [review]

File: common-design-patterns/most-requested.html (hunk 1 above)
Comment (delisma) [reply common-design-patterns/most-requested.html:2]: And the `breadcrumbs`

Comment (duboisp) [reply common-design-patterns/most-requested.html:2]: done

**Pull request closed without merging**